packages/core/src/components/section/section.tsx

```diff
@@ -109,7 +112,7 @@ export const Section: React.FC<SectionProps> = React.forwardRef((props, ref) =>
         title,
         ...htmlProps
     } = props;
-    const [isCollapsed, setIsCollapsed] = React.useState<boolean>(collapseProps?.defaultIsOpen ?? false);
+    const [isCollapsed, setIsCollapsed] = React.useState<boolean>(!collapseProps?.defaultIsOpen ?? false);
```

Comment:
The `!` turns `collapseProps?.defaultIsOpen` into a boolean. Is `?? false` needed if the left side will never be `undefined` or `null`?

It is a bit verbose, but I think `collapseProps?.defaultIsOpen ?? true` conveys the new `@default true` pretty well. Perhaps we should invert after performing that defaulting logic?

```suggestion
    const [isCollapsed, setIsCollapsed] = React.useState<boolean>(!(collapseProps?.defaultIsOpen ?? true));
```

Reply:
Agreed, that does make for more readable code! Updated the PR with the suggested changed and added a comment to the code to better explain the behavior. Also added a new test to assert the `defaultIsOpen={undefined}` case.

Reply:
Thanks!